recipes/pyls-mypy/LICENSE

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I assume the License file is not packaged in the sdist. Is this fixed upstream?

Reply:
The tar.gz doesn't contain the License file, but they have it in the repository https://github.com/tomv564/pyls-mypy/blob/master/LICENSE, should I remove that file from this PR?

Reply:
The filename should be included in the `MANIFEST.in` file upstream so it's included in future versions. Without it the tarball violates this line of the license:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.